snowshu/adapters/target_adapters/postgres_adapter/postgres_adapter.py

```diff
@@ -87,8 +86,11 @@ def __init__(self, replica_metadata: dict, **kwargs):
                                      f'-h {self._credentials.host} '
                                      f'-U {self._credentials.user} '
                                      f'-d {self._credentials.database}')
-        self.DOCKER_SHARE_REPLICA_DATA = f"cp -af $PGDATA/* {self.DOCKER_REPLICA_MOUNT_FOLDER}"  # noqa pylint: disable=invalid-name
-        self.DOCKER_IMPORT_REPLICA_DATA_FROM_SHARE = f"cp -R -f {self.DOCKER_REPLICA_MOUNT_FOLDER}/* $PGDATA/"  # noqa pylint: disable=invalid-name
+        self.DOCKER_SHARE_REPLICA_DATA = f"pg_dumpall -c -U {self._credentials.user} -p 9999 | " \
+                                         f"gzip > {self.DOCKER_REPLICA_MOUNT_FOLDER}/replica_dump.gz"  # noqa pylint: disable=invalid-name
```

Comment:
initially it was suggested to look into 
```
-F format
--format=format
Selects the format of the output. format can be one of the following:
```
https://www.postgresql.org/docs/12/app-pgdump.html

Reply:
Probably if we dump databases individually, but that adds more complexity for an unclear benefit

Reply:
I was asking about dumpall cmd